microsoft-edge/devtools-guide-chromium/experimental-features/focus-mode.md

```diff
@@ -15,7 +15,7 @@ Focus Mode replaces the main row of tabs with an **Activity Bar**, which is a co
 
 Focus Mode also provides a **Quick View** list, to open a second tool alongside the tool that's already selected in the **Activity Bar**.
 
-![DevTools undocked in its own window, with the Focus Mode experiment](media/focus-mode.png)
+![DevTools undocked in its own window, with the Focus Mode experiment](./focus-mode-images/focus-mode.png)
```

Comment:
* Start article by showing eg Todo Demo on left, DevTools docked (not undocked in sep window like presently).
* Expand Quick View panel so "Quick View" is visible.  
* Red box on Quick View.  
* Red box on Activity Bar.

gh rendered:
https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/focus-mode/microsoft-edge/devtools-guide-chromium/experimental-features/focus-mode.md#simplify-devtools-using-focus-mode